.travis/now

```diff
@@ -161,12 +162,27 @@ install () {
         <<<"export PYTHONPATH=\"$(_join ':' "${PY_PATH[@]}"):\$PYTHONPATH\";"
 }
 
-report_coverage () {
+_report_coverage () {
     coverage combine --append
     coverage xml --ignore-errors
     bash <(curl -s https://codecov.io/bash)
 }
 
+_report_error() {
+    # don't bail out on error
+    set +eu
+
+    printenv PATH PYTHONPATH
+    rose check-software
+    cat /tmp/sphinx-err*  # sphinx traceback
```

Comment:
Doesn't matter here, but should redirect error messages to `>&2` as a good practice.

Reply:
Doesn't make a jot of difference but done!